core/examples/luigi-sample-angular/src/assets/basicConfiguration.bundle.js

```diff
@@ -0,0 +1,93 @@
+/*!
```

Comment:
I think this file should not be in the repository as stated in .gitignore file in luigi-sample-angular folder. Same applies for core/examples/luigi-sample-angular/src/assets/extendedConfiguration.bundle.js and also for core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js, which was moved to another folder.

Reply:
done